CONTRIBUITING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to the Elastic Serverless Forwarder
+
+If you have a bugfix or new feature that you would like to contribute to
+elastic-serverless-forwarder, please find or open an issue about it first. Talk about what you would like to do. It may be that somebody is already working on it, or that there are particular issues that you should know about before implementing the change.
+
+We enjoy working with contributors to get their code accepted. There are many approaches to fixing a problem and it is important to find the best approach before writing too much code.
+
+## Running Elastic Serverless Forwarder locally
+
+We don't provide yet a tool for running Elastic Serverless Forwarder locally. A good first contribution would be to add such support.
```

Comment:
Do we have an issue for this? 😬

Reply:
I will add one! 👍

Reply:
https://github.com/elastic/elastic-serverless-forwarder/issues/87